### PR TITLE
Add required scopes to API endpoints

### PR DESCRIPTION
**Pull request by jenssegers**
The API documentation is missing information about required scopes for our endpoints.

### CODE REVIEW

**Review by WouterSioen — DISMISSED** [review]
What about related data? (F.e. customer coupled to a deal)

**Review by jdrieghe — DISMISSED** [review]
@WouterSioen with quite an interesting question here

File: apiary.apib [hunk 1]
```diff
@@ -2232,6 +2255,10 @@ Get a list of available tax rates.
                     + type: `department` (string)
                     + id: `182af0a8-3f68-409b-8941-cf8caf8f28a0` (string)
 
+## Payment Terms [/paymentTerms]
+
+*Required scopes: `invoices`*
```
Comment (stivni) [inline apiary.apib]: I think this should be in CRM... because you can set a paymentTerm per customer without having invoicing

**Comment by stivni** [comment]
@WouterSioen not an issue, as we don't return the data, but only a reference to it...
whenever we start side-loading, we will have to check if the user and/or the integration are allowed to

File: apiary.apib (hunk 1 above)
Comment (jdrieghe) [reply apiary.apib]: And what would you do with the paymentTerm besides create invoices?
I think the concept of a paymentTerm is only relevant for the invoicing module, so it's pointless to use it without the invoices scope.

**Review by steftrenson — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (jenssegers) [reply apiary.apib]: We can also drop this scope.

**Comment by jenssegers** [comment]
I removed the scope for payment terms.

**Review by WouterSioen — APPROVED** [review]

**Review by steftrenson — APPROVED** [review]

**Review by JordyDeGroeve — APPROVED** [review]

**Pull request merged**